lib/Importer/ONNXModelLoader.cpp

```diff
@@ -688,13 +683,29 @@ llvm::Error ONNXModelLoader::loadPool(const ONNX_NAMESPACE::NodeProto &op,
   ASSIGN_VALUE_OR_RETURN_ERR(pads, getPads(dict, kernels, strides, idimHW));
 
   Node *node = nullptr;
-  if (typeName == "MaxPool") {
+  if (op.output_size() > 1) {
+    if (typeName != "MaxPool") {
+      RETURN_ERR("Argmax output is only supported for MaxPool!",
+                 GlowErr::ErrorCode::MODEL_LOADER_UNSUPPORTED_OPERATOR);
+    }
+
     node = G_.createMaxPool(opName, tr, kernels, strides, pads);
+    auto *Res = G_.createTranspose(opName, NodeValue(node, 0), NHWC2NCHW);
```

Comment:
nit: We usually use `lowerCamelCase`, e.g. `res` here, `argmax` on next line, etc.

Reply:
Right, I have no idea why I named it like this here. :)